src/applications/vaos/covid-19-vaccine/components/ReviewPage.jsx

```diff
@@ -120,49 +120,48 @@ export default function ReviewPage() {
         </LoadingButton>
       </div>
       {submitStatus === FETCH_STATUS.failed && (
-        <div role="alert">
-          <AlertBox
+        <div className="info-alert">
```

Comment:
We do need to keep the `role="alert"` here, based on the accessibility review feedback. I think the idea is that we use them when we're using the va-alerts for feedback on an action (e.g. all the validation messages have an alert role).